### PR TITLE
Generate en_US.UTF-8 locale

### LINKED ISSUE #4889: perl: warning: Setting locale failed
### Qubes OS version:

4.0

### Affected component(s) or functionality:

Whonix

---

### Steps to reproduce the behavior:

in terminal:

`vrms`

### Expected or desired behavior:

normal output

### Actual behavior:

```
vrms 
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
        LANGUAGE = "",
        LC_ALL = (unset),
        LANG = "en_US.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to the standard locale ("C").

No non-free or contrib packages installed on host!  rms would be proud.
```

### General notes:

Doesn't only happen (with obscure example) `vrms` also with many other applications, users reported it happening with youtube-dl.

This randomly started happening. Some update must have introduced it. No Whonix code related to this was changed in years.

A "lazy" fix would be installing package `locales-all`. But disk space could be pushing it?

> `After this operation, 130 MB of additional disk space will be used.`

Anyone having this issue outside of Whonix?

During template build we have: https://github.com/QubesOS/qubes-builder-debian/blob/master/template_debian/distribution.sh#L354-L358

```
function updateLocale() {
    debug "Updating locales"
    chroot_cmd localedef -f UTF-8 -i en_US -c en_US.UTF-8
    chroot_cmd update-locale LANG=en_US.UTF-8
}
```

I wonder why this randomly stops working now?

### PR DESCRIPTION
**Pull request by DemiMarie**
Otherwise stuff breaks.

Fixes QubesOS/qubes-issues#4889.

### CODE REVIEW

**Comment by DemiMarie** [comment]
PipelineRetry

File: template_debian/distribution.sh [hunk 1]
```diff
@@ -358,6 +358,8 @@ EOF
 # ==============================================================================
 function updateLocale() {
     debug "Updating locales"
+    chroot_cmd sh -c 'printf "\\nen_US.UTF-8 UTF-8" >> /etc/locales.gen'
+    chroot_cmd locale-gen
```
Comment (marmarek) [inline template_debian/distribution.sh]: `locale-gen` is just a fancy wrapper around `localedef`. If adding the above is necessary, then no point in leaving the manual call below.

**Comment by rocodes** [comment]
Is this still planned to be merged? In 4.2.2-rc1, I see `locale` errors in the journal in debian minimal based qubes (for `qubes.VMShell-dom0 gtk-launch` and `qubes.StartApp+qubes-run-terminal-dom0`  for xfce-terminal) complaining about missing locale - looks like this would do the trick :)

**Comment by marmarek** [comment]
PipelineRetry

**Pull request merged**